libs/SalesforceSDKCore/SalesforceSDKCoreTests/SFUserAccountManagerPersisterTests.m

```diff
@@ -0,0 +1,381 @@
+/*
```

Comment:
This class seems like it's testing a lot of stuff that's not strictly related to account persistence.  Shouldn't this class be more specifically focused on testing account persistence?